src/Engine/MapController.java

```diff
@@ -56,8 +56,8 @@ public MapController(GameInstance game, MapView view, boolean initGame)
     isGameOver = false;
     nextSeekIndex = 0;
 
-    if( initGame )
-      // Start the first turn.
+    // Start the first turn (or the next one if loading a protected save.
+    if( initGame || myGame.requireInitOnLoad() )
```

Comment:
Why not just throw out `initGame` and make `currentTurnEnded` default to false?

Reply:
Because then it wouldn't call `startNextTurn()` to initialize the first turn when starting a new game... if I'm understanding you correctly.

Reply:
Sure it would. `currentTurnEnded` is false, so `requireInitOnLoad()` returns true.
The only call that uses this constructor is for loading a save from the main menu.

Reply:
`currentTurnEnded` defaults to false, yes, but that means `requireInitOnLoad()` returns false also. Plus, this constructor is called with `initGame==true` by the other `MapController` constructor.

Reply:
> `currentTurnEnded` defaults to false, yes, but that means `requireInitOnLoad()` returns false also. Plus, this constructor is called with `initGame==true` by the other `MapController` constructor.

...right. Derp.
So you should default it to *true*, so it effectively replaces the `initGame==true`.
We then need only one constructor, with the lower number of params.

Reply:
Yeah, I think that makes sense. Probably should have made that leap on my own. I blame after midnight and being distracted by making the config options screen scroll.